### DISCUSSION TITLE
Proxy whitelisting

### DISCUSSION THREAD

**Discussion by acharseth** [deployphp/deployer#3730] [General]
For setting up our staging environment we need to know what URLs should be whitelisted for running Deployer. We therefor checked what traffic is generated when running a deployment with Deployer and then see traffic to:
[deployer.org](https://deployer.org/)
[deployer.medv.io](https://deployer.medv.io/)
[packagist.org](https://packagist.org/)

Traffic to `packages.org` is natural (for the `composer`-part), but why do we see traffic to `deployer.org` and what is this?

**Comment by antonmedv** [comment]
Deployer uses those for internal telemetry. You can find a [env](https://github.com/deployphp/deployer/blob/a0a21e22a33860356d94bb14b67ecd37673bd1b8/src/Command/CommandCommon.php#L24) variable to disable it.

**Comment by acharseth** [comment]
Thanks! :-)
